# Message provides the relevant attributes specifying Dynamixel movement.
# Author: Bryan Butenhoff <syn7gerr@vt.edu>
Header header
std_msgs/Int64 moving_speed
std_msgs/Int64 torque_limit
std_msgs/Int64 goal_position